issue_25/get_seq.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Oct 14 11:25:08 2020
+
+@author: Bátora Dániel
+"""
+
+import os 
+from Bio import SeqIO
+from typing import Union
+
+
+
+def get_seq(filepath:str,organism:Union[str, int],temp_dir:str) -> str:
+    
+    """
+    Selects DNA sequence of the desired organims,
+    outputs it into a FASTA file 
+    
+    Args: 
+        filepath(str): Path to the original FASTA file 
+        organism(Union[str, int]): organism short name(str) or taxon id(int)
+        temp_dir(str): Path to the temporary directory
+        
+    Returns: 
+        output_path(str): Path to the output file saved in the temporary directory
+    
+    """
+    
+    
+    
+    if not os.path.isdir(temp_dir): 
+        os.mkdir(temp_dir)
+    output_path = os.path.join(temp_dir, organism + ".fasta")
+    reads = []
+    with open(filepath,  "r", encoding = "utf-8") as handle:
+        for record in SeqIO.parse(handle, "fasta"):
+            reads.append(record)
+    
+    hits = []
+    for record in reads:
+        if (isinstance(organism, str) and record.description.split("|")[3] == organism) or (isinstance(organism, int) and record.description.split("|")[4] == str(organism)):
+                hits.append(record)
+    SeqIO.write(hits, output_path, "fasta")
+    
+    return output_path
+        
+hits = get_seq(os.path.join(os.getcwd(), "transcripts.fasta"),  "avaga", "C:/github/htsinfer/data/temp_dir")
```

Comment:
This should go in the `infer()` function of the `htsinfer/infer_read_orientation.py` module

Reply:
Also, it's better to be explicit (makes it easier to read), so mention the argument names when calling a function, at least if it's more than one arg you are passing.